department-of-veterans-affairs/next-build
# VANB-18286: Changing from https to http for unblocking local build.

## Description
Closes #[18286](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18286).

## Testing done
Local build

## Screenshots


## QA steps
What needs to be checked to prove this works?  
Spin up a local for next-build
What needs to be checked to prove it didn't break any related things?  
Environments should work as expected
What variations of circumstances (users, actions, values) need to be checked?

```[tasklist]
```

## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
